scripts/bootstrap.sh.in

```diff
@@ -182,24 +248,51 @@ ${gpg_keys:+gpgkey=${gpg_keys%?}}
 EOF
 }
 
-install_salt_minion() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
+configure_apt_local_repository() {
+    local -r repo_name=$1
+    local -r repo_path="$BASE_DIR/packages/debian/$repo_name"
+
+    echo "deb [trusted=yes] file://$repo_path bionic $repo_name" \
+        >   /etc/apt/sources.list.d/"$repo_name".list
 
-    "$YUM" install "${yum_opts[@]}" salt-minion
 }
 
-install_genisoimage() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
+install_salt_minion() {
+    case "$OS" in
+        centos)
+            local -a yum_opts=(
+                '--assumeyes'
+                '--disablerepo=*'
+                '--enablerepo=metalk8s-*'
+            )
+
+            "$YUM" install "${yum_opts[@]}" salt-minion
+            ;;
+        ubuntu)
+            "$APT" install salt-minion
+            ;;
+        *)
+            die "Impossible to install Salt-minion";;
+    esac
+}
 
-    "$YUM" install "${yum_opts[@]}" genisoimage
+install_genisoimage() {
+    case "$OS" in
+        centos)
+            local -a yum_opts=(
+                '--assumeyes'
+                '--disablerepo=*'
+                '--enablerepo=metalk8s-*'
+             )
+
+            "$YUM" install "${yum_opts[@]}" genisoimage
+            ;;
+        ubuntu)
+            "$APT" install genisoimage
+            ;;
+        *)
+            die "Impossible to install genisoimage";;
```

Comment:
Same as above, wrong error message.
Maybe... instead of having this case everywhere, you could check once and for all that we support the current OS at the beginning of the script.